pkg/osquery/table/platform_tables.go

```diff
@@ -4,12 +4,16 @@ package table
 
 import (
 	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/osquery/tables/gsettings"
 	osquery "github.com/kolide/osquery-go"
 	"github.com/kolide/osquery-go/plugin/table"
 )
 
 // platformTables returns an empty set. It's here as a catchall for
 // unimplemented platforms.
 func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, currentOsquerydBinaryPath string) []*table.Plugin {
-	return []*table.Plugin{}
+	return []*table.Plugin{
+		gsettings.Settings(client, logger),
+		gsettings.Metadata(client, logger),
+	}
```

Comment:
This table won't build on windows, so this can't be here. We can probably skip this on darwin, so probably best to plumb in a `_linux` flavor here.

(If you think it's worth keeping on darwin, we can make another layer in here)

Reply:
funnily enough, I actually have `gsettings` installed on my mac (it was some homebrew dependency..). But I don't think we want to bother with this on darwin, it's more likely to lead to confusion than any actual benefit.